EventHorizon/Indicator/AuraIndicator.lua

```diff
@@ -123,6 +128,15 @@ function AuraIndicator:ApplyTicksAfter(start, stop, lastTick, duration)
 	-- lastTickProximiySec: Align the last added tick with proximity to the stop point of aura 
 	-- (cosmetics due to server internal jitter on duration extending effects)
 	local lastTickProximitySec = 0.2 
+
+	local duration = stop - start
+	if self.config.ticks and self.config.tickType == "count" and self.config.tickCount > 0 then		
+		self.numTicks = self.config.tickCount
+		self.snapInterval = duration / self.numTicks		
+	elseif self.config.ticks and self.config.tickType == "interval" and self.config.tickInterval > 0 then
+		self.snapInterval = self.config.tickInterval
+		self.numTicks = math.floor(duration/self.config.tickInterval)		
+	end
```

Comment:
Not needed and actually breaks some interactions.
However, this can be used to solve the Corruption hasted behavior.

The following was tested:
```
	-- Corruption edge case (the only refreshable hastable spell known)
	-- TODO(verill): Refactor in future if there are more cases like corruption
	local lastTick = lastTick
	if self.spellName == "Corruption" then
		-- remove any ticks after start of new duration
		self:RemoveTicksAfter(start)
		lastTick = self.ticks[#self.ticks].start

		-- Corruption is a hasted refreshable spell and we break the immutability of the dot instance because of that
		local duration = stop - start
		if self.config.ticks and self.config.tickType == "count" and self.config.tickCount > 0 then		
			self.numTicks = self.config.tickCount
			self.snapInterval = duration / self.numTicks		
		elseif self.config.ticks and self.config.tickType == "interval" and self.config.tickInterval > 0 then
			self.snapInterval = self.config.tickInterval
			self.numTicks = math.floor(duration/self.config.tickInterval)		
		end	
	end
```

Reply:
fixed in latest commit